.github/workflows/auto-format.yml

```
@@ -0,0 +1,28 @@
+name: Auto-format
+
+on:
+  push:
+    branches: ['development']
+  pull_request:
+    branches: ['development']
```

Shouldn't this be for all branches and only in PRs? I also wonder if this will get too annoying if you're working on a draft PR and start needing to pull commits, but I guess we'll find out 😂 
```suggestion
  pull_request:
    branches: *
```

---

Ok. I really have no idea how this part of the yaml works. But, when I had this it worked on my fork with a branch that wasn't development. Does what you suggested mean a pull request into any branch and mine say a pull request into only development? Also,  does pull_request mean it runs anytime there is a change to a pull request.. I was thinking the "Push" was there to also run on new pushes to a pull request?

---

Basically what you had means (if I'm not wrong) that this job will run for every commit pushed to `development`, and for every PR created that target `development` (and any push to those PRs' branches). If you create PRs that sit on top of other PRs (like we usually do), those won't trigger this job.

Looking at [ci.yml](https://github.com/desktop/desktop/blob/426c834a117311f255cc303de28ac9c041c1242b/.github/workflows/ci.yml#L3-L9) I think we only need `pull_request:` (with no `branches:` line at all) and that's it.